packages/notification-services-controller/CHANGELOG.md

```diff
@@ -7,11 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0]
```

Comment:
If no features are being added and the bug fixes are the only new changes, maybe this should be a patch release?

```suggestion
## [0.1.1]
```

Reply:
OH, yes we can make this a patch release, no probs!

Reply:
TY for the comments, I've updated the changelog and packages for minor version.